bindings/guile/gnc-kvp-guile.cpp

```diff
@@ -2,6 +2,7 @@
 #include <kvp-frame.hpp>
 #include <libguile.h>
 #include <numeric>
+#include <stdint.h>
```

Comment:
stdint.h is the C header. Please use cstdint in C++ files.

Reply:
```
cstdint -> std::<intX_t>
stdint.h -> <intX_t>
```
one problem of gnucash is that it relies on `<cstdint>` providing unqualified `<intX_t>` which is not guaranteed by the standard. `<stdint.h>` guarantees unqualified types.

Reply:
updated